### PR TITLE
✨Add ability to change LLEMU text/background color.

### LINKED ISSUE #343: Add Background Color Changing to LLEMU
From time to time, a member of the dev team is approached and asked to bake a custom version of the Kernel where the background color of LLEMU (default: lime green) is changed to something more suitable to their team color. To prevent unnecessary work for us, we should implement a function that changes the color for the user.

### PR DESCRIPTION
**Pull request by omegaStag**
#### Summary:
Added function to change the LCD background color (takes in ``lv_color_t`` type) and an overload for the function on the C++ level (takes in both ``lv_color_t``, and three ``uint8_t`` values representing RGB values).

#### Motivation:
So that users can change the LCD background color.

##### References (optional):
closes #343 

#### Test Plan:
Use function and see if it works to change the background color.

### CODE REVIEW

File: src/display/llemu.c [hunk 1]
```diff
@@ -309,3 +309,8 @@ uint8_t lcd_read_buttons(void) {
 	}
 	return _lcd_read_buttons(_llemu_lcd);
 }
+
+void lcd_set_background_color(lv_color_t color) {
+	screen_style.body.main_color = color;
+	screen_style.body.grad_color = color;
+}
```
Comment (WillXuCodes) [inline src/display/llemu.c:316]: Add a newline to the end of this file

File: include/pros/llemu.h [hunk 2]
```diff
@@ -225,6 +225,14 @@ bool lcd_register_btn2_cb(lcd_btn_cb_fn_t cb);
  */
 uint8_t lcd_read_buttons(void);
 
+/**
+ * Changes the color of the LCD background to a provided color in type
+ * lv_color_t.
+ *
+ * \return void
+ */
```
Comment (WillXuCodes) [inline include/pros/llemu.h:233]: Needs documentation for parameter

File: include/pros/llemu.hpp [hunk 3]
```diff
@@ -193,6 +193,23 @@ void register_btn2_cb(lcd_btn_cb_fn_t cb);
...
+ */
+void set_background_color(lv_color_t color);
+
+/**
+ * Changes the color of the LCD background to a provided color expressed in RGB
+ * form, with three values of type uint8_t.
+ * 
+ * \return void
+ */
+void set_background_color(std::uint8_t r, std::uint8_t g, std::uint8_t b);
```
Comment (WillXuCodes) [inline include/pros/llemu.hpp:211]: Also needs documentation for parameter

**Review by WillXuCodes — CHANGES_REQUESTED** [review]

**Comment by Unionjackjz1** [comment]
thank you <3

File: src/display/llemu.cpp [hunk 4]
```diff
@@ -50,5 +50,14 @@ void register_btn2_cb(lcd_btn_cb_fn_t cb) {
 std::uint8_t read_buttons(void) {
 	return lcd_read_buttons();
 }
+
+void set_background_color(lv_color_t color) {
+	lcd_set_background_color(color);
+}
+
+void set_background_color(uint8_t r, uint8_t g, uint8_t b) {
+	lcd_set_background_color(lv_color_t{r,g,b,0xff});
```
Comment (WillXuCodes) [inline src/display/llemu.cpp:59]:
```suggestion
	lcd_set_background_color(LV_COLOR_MAKE(r, g, b));
```

Should prob have this use the macro as well, just so the config for alpha works.

**Review by WillXuCodes — CHANGES_REQUESTED** [review]

File: include/pros/llemu.hpp [hunk 5]
```diff
@@ -193,6 +193,38 @@ void register_btn2_cb(lcd_btn_cb_fn_t cb);
...
+ * 
+ * \return void
+ */
+void set_background_color(lv_color_t color);
+
+/**
+ * Changes the color of the LCD background to a provided color expressed in RGB
+ * form, with three values of type uint8_t.
+ * 
+ * \param r
```
Comment (WillXuCodes) [inline include/pros/llemu.hpp:212]: Need proper tabbing for comments

**Review by WillXuCodes — APPROVED** [review]
Approved, tested with Hardware.

**Pull request merged**